yamls/kafka/bridge/route.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: route.openshift.io/v1
```

Comment:
I will try this but it may not work. I think the Host needs to be left blank so that the cluster ingress can resolve the domain names automatically